jest-setup.js

```diff
@@ -0,0 +1,5 @@
+import '@testing-library/jest-dom';
```

Comment:
It it a common convention to put it in root folder or should/could it live somewhere else, like `/test` (although we have some CSS metrics tests there I think)

Reply:
This file is usually placed in the root folder along with `package.json`, eslint config etc.